src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(({ id }) => id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title }) => (
+            <li
+              className={id === activeTab.id ? 'is-active' : ''}
```

Comment:
- You compare `id === activeTab.id` twice, create a variable
- To set classes conditionally use `classnames` or `clsx` library 
https://www.npmjs.com/package/clsx
https://www.npmjs.com/package/classnames